ppl/archive/chunk/reader.go

```diff
@@ -0,0 +1,59 @@
+package chunk
+
+import (
+	"context"
+	"errors"
+	"io"
+
+	"github.com/brimsec/zq/pkg/iosrc"
+	"github.com/brimsec/zq/pkg/nano"
+	"github.com/brimsec/zq/ppl/archive/seekindex"
+	"github.com/brimsec/zq/zqe"
+)
+
+type Reader struct {
+	io.Reader
+	io.Closer
+	TotalSize int64
+	ReadSize  int64
+}
+
+// newChunkReader returns an io.ReadCloser for this chunk. If the chunk has a seek
```

Comment:
Would you update the comment here, both for the method name & the return type it mentions?

Reply:
Whoops, yeah I'll check through the rest of the comments as well.